airbyte-cdk/python/airbyte_cdk/sources/abstract_source.py

```diff
@@ -130,6 +136,10 @@ def read(
 
         logger.info(f"Finished syncing {self.name}")
 
+    @property
+    def availability_strategy(self) -> Optional[AvailabilityStrategy]:
+        return None
```

Comment:
TODO: make default, need to mock the `check_availability` method in a lot of tests that break